### PR TITLE
Add Ansible DK 1.2.0

### PR DESCRIPTION
**Pull request by filler**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

Just like the Chef Development Kit (DK), there is an omnibus-based DK
for Ansible maintained by OmniTI.  This adds that.

### CODE REVIEW

File: Casks/ansible-dk.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'ansible-dk' do
+  version '1.2.0-3'
+  sha256 '70fe9e4b8f27e8961c992de3ed1e30bb39c43319af28aae73c177f9530352a49'
+
+  url 'https://github.com/omniti-labs/ansible-dk/releases/download/1.2.0/ansible-dk-1.2.0-3.dmg'
```
Comment (reitermarkus) [inline Casks/ansible-dk.rb]: This needs an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (filler) [inline Casks/ansible-dk.rb]: A-ha.  TIL!

Bumped.  Thanks.

File: Casks/ansible-dk.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  appcast 'https://github.com/omniti-labs/ansible-dk/releases.atom'
+  checkpoint 'e4ff73165aeb8fbb73bf77422ce1ee68c80e8569d695ece740865aa121fce931'
```
Comment (reitermarkus) [inline Casks/ansible-dk.rb]: Forgot the comma and indentation here.

```
  appcast 'https://github.com/omniti-labs/ansible-dk/releases.atom',
          checkpoint: 'e4ff73165aeb8fbb73bf77422ce1ee68c80e8569d695ece740865aa121fce931'

```

Comment (filler) [inline Casks/ansible-dk.rb]: Doh!  Updated.  :)

**Comment by reitermarkus** [comment]
Your pull request was merged directly in 3db9095a76bfd68be5afd0b57901d750870d63b8 to keep commit history cleaner. It needed some changes which were made in 3db9095a76bfd68be5afd0b57901d750870d63b8. Thank you for your contribution!

**Comment by filler** [comment]
Yay! Thanks!

**Pull request closed without merging**